internal/permissions/permission_store.go

```diff
@@ -30,110 +27,176 @@ type Store interface {
 	ListReadPermissionsByLexicon(owner string) (map[string][]string, error)
 }
 
-type casbinStore struct {
-	enforcer *casbin.Enforcer
-	adapter  persist.Adapter
+type sqliteStore struct {
+	db *gorm.DB
 }
 
-//go:embed model.conf
-var modelStr string
+var _ Store = (*sqliteStore)(nil)
 
-func NewStore(adapter persist.Adapter, autoSave bool) (Store, error) {
-	m, err := model.NewModelFromString(modelStr)
-	if err != nil {
-		return nil, err
-	}
-	enforcer, err := casbin.NewEnforcer(m, adapter)
+// Permission represents a permission entry in the database
+type Permission struct {
```

Comment:
thinking on the naming here ... is this a `Permission` or `Policy`? usually allow / deny go with policies but the grantee-owner-object feels lmore like permission